.github/workflows/ci_ubuntu.yml

```diff
@@ -15,9 +15,10 @@ jobs:
 
     services:
       minio:
-        image: lazybit/minio:latest
+        image: minio/minio:latest
```

Comment:
@LeoQuote - What was the main reason you used lazy bit/minio vs. the official image?

I'll see if I just need to pass more options. I'd like to use the official at it seems more maintained.

Reply:
the official minio image has an entrypoint `minio` while this command alone cannot start a minio server, and github action does not support adding argument or override entrypoint for image, I have to use the `lazybit/minio:latest`  image to get minio running.